crates/optimism/flashblocks/src/service.rs

```diff
@@ -234,15 +243,15 @@ where
                 // built a new pending block
                 this.current = Some(new_pending.clone());
                 this.rebuild = false;
-                trace!(parent_hash=%new_pending.block().parent_hash(), block_number=new_pending.block().number(), flash_blocks=this.blocks.count(), elapsed=?now.elapsed(), "Built new block with flashblocks");
+                trace!(target: "flashblocks", parent_hash=%new_pending.block().parent_hash(), block_number=new_pending.block().number(), flash_blocks=this.blocks.count(), elapsed=?now.elapsed(), "Built new block with flashblocks");
                 return Poll::Ready(Some(Ok(Some(new_pending))));
             }
             Ok(None) => {
                 // nothing to do because tracked flashblock doesn't attach to latest
             }
             Err(err) => {
                 // we can ignore this error
-                debug!(%err, "failed to execute flashblock");
+                debug!(target: "flashblocks", %err, "failed to execute flashblock");
```

Comment:
With this change, the `target` would be inconsistent throughout the `reth_optimism_flashblocks` crate.

I think it's fine to keep the default.

Then you can reliably use `RUST_LOG=reth_optimism_flashblocks=trace,info`